src/tests/Common/CLRTest.Jit.targets

```diff
@@ -231,30 +234,34 @@ IF NOT DEFINED DoLink (
       <BashDisasmListOutputFile Condition="'$(HasBashDisasmCheck)' == 'true'">$(scriptPath)__jit_disasm_list.out</BashDisasmListOutputFile>
 
       <BashCLRTestPreCommands Condition="'$(HasBashDisasmCheck)' == 'true'"><![CDATA[
-@(DisasmCheckFiles -> '  dotnet $CORE_ROOT/SuperFileCheck/SuperFileCheck.dll --csharp-list-method-names "%(Identity)" --allow-unused-prefixes --check-prefixes=CHECK,$(TargetArchitecture.ToUpperInvariant()),$(TargetArchitecture.ToUpperInvariant())-$(TargetOS.ToUpperInvariant()) > "$(BashDisasmListOutputFile)"
-  ERRORLEVEL=$?
-  export COMPlus_JitDisasm=`cat $(BashDisasmListOutputFile)`
-  export COMPlus_JitDiffableDasm=1
-  export COMPlus_JitStdOutFile=$(BashDisasmOutputFile)
-  if [[ $ERRORLEVEL -ne 0 ]]
-  then
-    echo EXECUTION OF FILECHECK - FAILED $ERRORLEVEL
-    exit 1
-  fi', '%0a')
-]]>
-      </BashCLRTestPreCommands>
-
-      <BashDisasmCheckPostCommands></BashDisasmCheckPostCommands>
-      <BashDisasmCheckPostCommands Condition="'$(HasBashDisasmCheck)' == 'true'"><![CDATA[
-if [[ -n $COMPlus_JitDisasm ]]; then
-  @(DisasmCheckFiles -> '  dotnet $CORE_ROOT/SuperFileCheck/SuperFileCheck.dll --csharp "%(Identity)" --allow-unused-prefixes --check-prefixes=CHECK,$(TargetArchitecture.ToUpperInvariant()),$(TargetArchitecture.ToUpperInvariant())-$(TargetOS.ToUpperInvariant()) --dump-input-context 25 --input-file "$(BashDisasmOutputFile)"
+if [[ ( -z "$COMPlus_JitStress" ) && ( -z "$COMPlus_JitStressRegs" ) && ( -z "$COMPlus_TailcallStress" ) && ( "$COMPlus_TieredPGO" != "1" ) ]]; then
```

Comment:
These need to check the DOTNET_ variants as well.

Reply:
If that's the case - then we need to update all the existing checks (outside of our FileCheck changes) that use `COMPlus_`. 

As I've mentioned before, these checks are from when `JitOptimizationSensitive` is set to `true` - so it is probably 'ok' that we use `COMPlus_` here.